api/v3/licensemaster_types.go

```diff
@@ -55,10 +55,10 @@ type LicenseMasterStatus struct {
 
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
-// LicenseMaster is the Schema for a Splunk Enterprise license manager.
+// LicenseMaster is the Schema for a Splunk Enterprise license master.
```

Comment:
These are to avoid conflicts between the two CRDs. In a few cases, we had to move the master back to allow the new manager labels to be unique.  We chose to roll them back in the *master* files since they'll eventually be removed.